contracts/seed/Seed.sol

```diff
@@ -559,8 +560,6 @@ contract Seed {
                 "Seed: should transfer seed tokens to refund receiver"
             );
         } else {
-            uint256 seedAmountRequired = (hardCap * PRECISION) / price;
```

Comment:
This fix doesn't correspond to the described issue, does it fix the thing you wanted to fix?

Reply:
After solving merge conflicts price and fee were removed from the global variables part, also seedAmountRequired and feeAmountRequired now are globally defined in the initialization part, so these two rows after merging other branches just started to be an error reason. They are not related to the current issue, but with them an error occurred